sentry/src/main/java/io/sentry/SendFireAndForgetEnvelopeSender.java

```diff
@@ -21,8 +21,8 @@ public SendFireAndForgetEnvelopeSender(
 
   @Override
   public @Nullable SendCachedEnvelopeFireAndForgetIntegration.SendFireAndForget create(
-      final @NotNull IHub hub, final @NotNull SentryOptions options) {
-    Objects.requireNonNull(hub, "Hub is required");
+      final @NotNull IScopes scopes, final @NotNull SentryOptions options) {
+    Objects.requireNonNull(scopes, "Hub is required");
```

Comment:
Message should refer to `Scopes` rather than `Hub`

Reply:
Changed in https://github.com/getsentry/sentry-java/pull/3366